### ISSUE TITLE
medium.com

### ISSUE THREAD

**Issue by bug-rules** [uBlockOrigin/uAssets#6015]
`https://itnext.io/structuring-your-sass-projects-c8d41fa55ed4`

floating bar (visible after open few other articles?) maybe no blocked by AdGuard / EasyList:

![obraz](https://user-images.githubusercontent.com/43518394/62012413-3128c400-b186-11e9-9a37-442210a676d8.png) |
--- |

```
##body > div > div > div:has(h4:has-text(/^Already have an account/)):matches-css(position: fixed)
```

or

```
##h4:has-text(/^Already have an account/):nth-ancestor(6):matches-css(position: fixed)
```
Now added new `div` and removed `[aria-modal="true"]` attrib.

**Comment by bug-rules** [comment]
@DandelionSprout?

**Comment by DandelionSprout** [comment]
Since the banner claims to offer more content in exchange for the login, this is a lower priority than usual as far as I am personally concerned. It remains to be seen what the actual uAssets team will think about it, however.

Alternately, in the very unlikely scenario that the banner is actually lying about providing one additional article, then I'd deal with it big time.

**Comment by DandelionSprout** [comment]
Additionally, it deserves mention that a fix for this could've been applied to several hundred Medium-based domains, and not just `itnext.io`.

**Comment by mapx-** [comment]
or
`itnext.io##[class*="svgIcon-use"]:nth-ancestor(8)`

**Comment by okiehsch** [comment]
> what the actual uAssets team will think about it

I think it can be fixed in `annoyances` it is a fairly big floating bar and it blocks content. 
We have added filters for similar issues.

**Comment by yourduskquibbles** [comment]
@mapx- `##[class*="svgIcon-use"]:nth-ancestor(8)` is over aggressive and removes the page header title info as well as the annoyance bar. See screenshots below:

![mediumtest3](https://user-images.githubusercontent.com/22258847/62053098-ed18dc00-b1e4-11e9-89de-73aa8255f684.png)


![mediumtest](https://user-images.githubusercontent.com/22258847/62052986-a7f4aa00-b1e4-11e9-9f6e-12f955b15351.png)


I think this is better filter `###root > div > div:has-text(Get one more story in your member)`

![mediumtest2](https://user-images.githubusercontent.com/22258847/62053050-cfe40d80-b1e4-11e9-8b95-9ac834cd4238.png)

Can someone else test and confirm? Only problem may be if text is region/language specific it won't be applicable when user is not in English locale. 

Medium Test Link: `https://forge.medium.com/how-do-i-motivate-my-stay-at-home-husband-to-go-back-to-work-f26db62ff43b`

itnext Test Link: `https://itnext.io/structuring-your-sass-projects-c8d41fa55ed4`

**Comment by mapx-** [comment]
well, ..feel free to remove it and add a working (correct) filter, thanks
